### ISSUE TITLE
Feature name restricted to ASCII_ALPHANUMERIC

### ISSUE THREAD

**Issue by sebpuetz** [danieldk/dpar#37]
The feature definition file is restricted to ASCII_ALPHANUMERIC feature names. I don't know if that's wanted, but there are probably cases where people (_me_) have features with underscores or some other non-ascii-alphanumeric chars. 

Quick fix for this would be replace ASCII_ALPHANUMERIC in the grammar definition file by a newly defined `char` rule:

```
feature_name = ${ char+ }
char = _{ !(WHITESPACE | "|" | ":" ) ~ ANY }
```

or a more defensive change to `feature_name` by whitelisting non-ascii-alphanumeric letters explicitly

```
feature_name = ${ (ASCI_ALPHANUMERIC | "_" | "-" )+ }
```

**Comment by danieldk** [comment]
I am more in favor of the latter. Feel free to do a PR!

Note though that the current master version uses a lot of memory (#36 ) , so it's best to write some unit tests for these changes, so that they can be tested without having to run the full parser.

**Comment by sebpuetz** [comment]
I'll do that later or tomorrow!

Something slightly related to this: we could/should provide an example feature definition file alongside the example config under `dpar-utils/testdata/`.

**Comment by sebpuetz** [comment]
Closed by #38
